lecture/tests.py

```
@@ -1,3 +1,92 @@
+        self.assertEqual(self.sample_lecture.module, self.sample_module)
+        self.assertEqual(self.sample_lecture.order, 1)
+
+    def test_slug_generation_on_save(self):
+        self.assertEqual(self.sample_lecture.slug, slugify(LECTURE_TITLE))
+
+    def test_str_representation(self):
+        self.assertEqual(str(self.sample_lecture), f"{self.sample_course.title} - {LECTURE_TITLE}")
+
+
+class LecturePDFModelTest(TestCase):
+    def setUp(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)